src/amo/components/OverallRating.js

```diff
@@ -54,21 +54,23 @@ export class OverallRatingBase extends React.Component {
           <fieldset>
             <legend>{prompt}</legend>
             <div className="OverallRating-choices">
-              <button value={5} onClick={this.onClickRating}
-                className="OverallRating-choice OverallRating-rating-5" id="OverallRating-rating-5">
-                {/* L10n: This should be a very short phrase for layout reasons, if possible. */}
-                {i18n.gettext('Love it!')}
-              </button>
-              <button value={3} onClick={this.onClickRating}
-                className="OverallRating-choice OverallRating-rating-3" id="OverallRating-rating-3">
-                {/* L10n: This should be a very short phrase for layout reasons, if possible. */}
-                {i18n.gettext("It's OK")}
-              </button>
-              <button value={1} onClick={this.onClickRating}
-                className="OverallRating-choice OverallRating-rating-1" id="OverallRating-rating-1">
-                {/* L10n: This should be a very short phrase for layout reasons, if possible. */}
-                {i18n.gettext('Huh?')}
-              </button>
+              <span className="OverallRating-star-group">
+                <button value={1} onClick={this.onClickRating}
```

Comment:
I thought about it for a minute but since there are only five it's probably ok to leave them. Generated tests always make me nervous because a bug could cause missed tests.